contracts/interchain-token/tests/test.rs

```diff
@@ -186,58 +101,36 @@ fn minter_test() {
 
     let token = create_token(&env, &owner, &minter1);
```

Comment:
maybe change `create_token` into a `setup_token` method that creates and returns the env, owner, minter etc. to reduce setup duplication